tests/foo.rs

```diff
@@ -0,0 +1,28 @@
+use cranelift_codegen::settings;
```

Comment:
If we do keep it as a unit test, perhaps this should be `instantiate_empty_module.rs` instead of `foo.rs`?

Reply:
Oh I see this was an active question above as where this should go, hence the name.  👍